lsusb.c

```diff
@@ -288,6 +289,30 @@ static void dump_device(
 		read_sysfs_prop(serial, sizeof(serial), sysfs_name, "serial");
 	}
 
+	speed = libusb_get_device_speed(dev);
+	const char *negotiated_speed = NULL;
```

Comment:
Shouldn't this go to the top of the function as well?  Including variable definitions in the middle of the function is a newer C functionality, might break on older compilers, right?